assets/_scss/components/_forms.scss

```diff
@@ -129,3 +152,14 @@ input.usa-input-medium {
   font-style: italic;
   font-weight: normal;
 }
+
+.usa-input-buttons-inline {
```

Comment:
It seems like this could be potentially abstracted further into .usa-input-inline? What do you think @msecret @blacktm Is there anything unique to it being a button?